easybuild/easyconfigs/l/libxc/libxc-3.0.0-foss-2016b.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'ConfigureMake'
+
+name = 'libxc'
+version = '3.0.0'
+
+homepage = 'http://www.tddft.org/programs/octopus/wiki/index.php/Libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.tddft.org/programs/octopus/down.php?file=libxc/']
+
+configopts = 'FC="$F77" FCFLAGS="$FFLAGS" --enable-shared --enable-fortran'
+
+# From the libxc mailing list
+# To summarize: expect less tests to fail in libxc 2.0.2, but don't expect
+# a fully working testsuite soon (unless someone wants to volunteer to do
+# it, of course  ) In the meantime, unless the majority of the tests
+# fail, your build should be fine.
+runtest = 'check'
```

Comment:
@boegel @migueldiascosta I n oticed that there is now a [libxc-3.0.0-GCC-5.4.0-2.26](https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/l/libxc/libxc-3.0.0-GCC-5.4.0-2.26.eb) in master, which could perhaps be used for foss-2016b in general. In any case I borrowed the` toolchainopts = {'lowopt': True}`  from that (see a2ebc76dd1ea43e5821adc240ec9790ac9e2469b ) to hopefully pass the libxc tests 😅